electrumsv/network_support/mapi.py

```diff
@@ -218,44 +227,56 @@ def validate_json_envelope(json_response: JSONEnvelope) -> None:
             raise ValueError("MAPI signature invalid")
 
 
-async def broadcast_transaction(tx: "Transaction", server: "NewServer",
-        credential_id: Optional["IndefiniteCredentialId"]) -> None:
-    server_state = server.api_key_state[credential_id]
-    server_state.record_attempt()
+async def broadcast_transaction_mapi_simple(tx: "Transaction", server: "NewServer",
+        credential_id: Optional["IndefiniteCredentialId"], peer_channel: Optional[PeerChannel]=None,
+        merkle_proof: bool=False, ds_check: bool=False) -> BroadcastResponse:
```

Comment:
I think we always want merkle proof and double spend check and to be passing a peer channel. Get rid of all the options and make the standard call the only one.